wire-ios-request-strategy/Sources/Request Strategies/Conversation/ConversationRequestStrategy.swift

```diff
@@ -373,18 +362,6 @@ extension ConversationRequestStrategy: ZMUpstreamTranscoder {
     public func updateUpdatedObject(_ managedObject: ZMManagedObject,
                                     requestUserInfo: [AnyHashable: Any]? = nil,
                                     response: ZMTransportResponse, keysToParse: Set<String>) -> Bool {
-
-        guard
```

Comment:
@typfel can you confirm we don't need to parse the payloads here with ConversationEventProcessor?

Reply:
We still need to parse the payloads here with `ConversationEventProcessor `. This was accidentally removed when replacing the `ZMEventConsumer` with `ZMAsyncEventConsumer`?